packages/ui/src/form-elements/map/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {FC} from "react";
+import {Paragraph, FormLabel, FormFieldDescription, FormField} from "@utrecht/component-library-react";
+import './map.css';
+
+export type MapProps = {
+    title: string;
+    description: string;
+    fieldKey: string;
+    fieldRequired: boolean;
+}
+
+const MapField: FC<MapProps> = ({
+    title,
+    description,
+    fieldKey,
+    fieldRequired= false,
+    onChange
+}) => {
+    const randomID = Math.random().toString(36).substring(2, 15) + Math.random().toString(36).substring(2, 15);
+
+    // TODO: Get map value when setting a marker. The function to retrieve the value is not yet implemented
```

Comment:
De Editormap zit nu in de main. Heeft dit component de handler waarmee je de value op kan halen? Of moet Niels dit nog aanmaken?